xml/System.Diagnostics/ProcessStartInfo.xml

```diff
@@ -1147,6 +1147,7 @@ p.WaitForExit();
   
  ]]></format>
         </remarks>
+        <exception cref="T:System.PlatformNotSupportedException">An attempt to set the value on Universal Windows Platform (UWP) apps occurs.</exception>
```

Comment:
The way that the code behaves is that it throws in UWP if the code attempts to set the value to true, when the value is set to false the code works fine, see https://github.com/dotnet/corefx/commits/master/src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Uap.cs

Reply:
Good catch @pjanotti! It was my bad!